docs/running-on-kubernetes.md

```diff
@@ -44,8 +57,8 @@ are set up as described above:
       --kubernetes-namespace default \
       --conf spark.executor.instances=5 \
       --conf spark.app.name=spark-pi \
-      --conf spark.kubernetes.driver.docker.image=registry-host:5000/spark-driver:latest \
-      --conf spark.kubernetes.executor.docker.image=registry-host:5000/spark-executor:latest \
+      --conf spark.kubernetes.driver.docker.image=kubespark/spark-driver:0.1.0-alpha.1 \
```

Comment:
Do we have a schema for the docker image tags? I think we may want include to the spark version, k8s version in the tag as well. 

Also we may want to use a more stable tag (e.g. 0.1-alpha, 0.2-stable) to avoid updating this file too often.